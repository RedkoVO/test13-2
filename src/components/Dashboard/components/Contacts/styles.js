export default () => ({
  root: {
    paddingTop: '35px'
  },
  title: {
    fontSize: '37px',
    fontWeight: 700,
    color: '#5e7cff',
    textAlign: 'center',
    marginBottom: '0px',
    textDecoration: 'none',
    margin: '0 auto',
    paddingBottom: '35px'
  },
  wrInfo: {
    display: 'flex',
    alignItems: 'center',
    marginBottom: '35px'
  },
  userPhoto: {
    height: '50px',
    width: '50px',
    display: 'block',
    borderRadius: '50%',
    boxShadow: '0px 5px 10px 0px rgba(0,0,0,0.1)',
    margin: '0 15px',

    '& img': {
      height: '100%',
      width: '100%',
      borderRadius: '50%'
    }
  },
  copyIcon: {
    width: '17px',
    height: '17px',
    cursor: 'pointer',
    marginLeft: '10px'
  }
})
