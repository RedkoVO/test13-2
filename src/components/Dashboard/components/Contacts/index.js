import React from 'react'
import withStyles from '@material-ui/core/styles/withStyles'

import Avatar from '../../../../assets/images/user.jpg'
import CopyIcon from '../../../../assets/images/svg/copy.svg'

import styles from './styles'

const Contacts = ({ classes }) => {
  return (
    <section id="contacts">
      <div className={classes.root}>
        <div className={classes.title}>Contacts</div>
        <div className={classes.content}>
          <div className={classes.wrInfo}>
            <div className={classes.userPhoto}>
              <img src={Avatar} alt="" />
            </div>
            <div className={classes.info}>
              <div>Mike Grinch</div>
              <div>
                0x000000000000000000000000000000000000000000000000000000000
                <img className={classes.copyIcon} src={CopyIcon} alt="" />
              </div>
            </div>
          </div>

          <div className={classes.wrInfo}>
            <div className={classes.userPhoto}>
              <img src={Avatar} alt="" />
            </div>
            <div className={classes.info}>
              <div>Clarck Malkolm</div>
              <div>
                0x000000000000000000000000000000000000000000000000000000000
                <img className={classes.copyIcon} src={CopyIcon} alt="" />
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
  )
}

Contacts.propTypes = {}

export default withStyles(styles)(Contacts)
