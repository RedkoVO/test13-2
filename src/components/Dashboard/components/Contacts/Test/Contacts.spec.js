import * as React from 'react'
import renderer from 'react-test-renderer'

import Contacts from '../'

it('renders correctly', () => {
  const tree = renderer.create(<Contacts />).toJSON()
  expect(tree).toMatchSnapshot()
})
