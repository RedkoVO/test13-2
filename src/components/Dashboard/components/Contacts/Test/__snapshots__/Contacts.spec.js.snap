// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  id="contacts"
>
  <div
    className="Contacts-root-1"
  >
    <div
      className="Contacts-title-2"
    >
      Contacts
    </div>
    <div>
      <div
        className="Contacts-wrInfo-3"
      >
        <div
          className="Contacts-userPhoto-4"
        >
          <img
            alt=""
            src="user.jpg"
          />
        </div>
        <div>
          <div>
            Mike Grinch
          </div>
          <div>
            0x000000000000000000000000000000000000000000000000000000000
            <img
              alt=""
              className="Contacts-copyIcon-5"
              src="copy.svg"
            />
          </div>
        </div>
      </div>
      <div
        className="Contacts-wrInfo-3"
      >
        <div
          className="Contacts-userPhoto-4"
        >
          <img
            alt=""
            src="user.jpg"
          />
        </div>
        <div>
          <div>
            Clarck Malkolm
          </div>
          <div>
            0x000000000000000000000000000000000000000000000000000000000
            <img
              alt=""
              className="Contacts-copyIcon-5"
              src="copy.svg"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
